Modules/Exercise05/Tests/Unit/Services/OrderServiceTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Modules\Exercise05\Tests\Unit\Services;
+
+use Modules\Exercise05\Services\OrderService;
+use Tests\TestCase;
+
+class OrderServiceTest extends TestCase
```

Comment:
Nên test thêm các case `=` nữa.
Hiện tại những case so sánh `>=` thì anh chỉ test case `>` nhưng case `=` thì hổng có test nha

Reply:
Trong provider này nên thêm có 1 case nữa check độ chính xác tới 2 chữ số thập phân vì cái round nó có thêm tham số thứ 2 là `2`
```
$infoBill['price'] = round(($detailOrder['price'] * 80) / 100, 2);
```
Cái này nice to have :smile:

Reply:
https://github.com/truongbt-0855/php-unit-test-homework/blob/eff93c32c243259ce19c1f76868aa9ffed02595c/Modules/Exercise05/Services/OrderService.php#L14

hàm này làm gì có cái >= đâu e nhỉ ?